R/zzz.R

```diff
@@ -116,6 +116,17 @@ move_env_elements(RPolarsExpr, pl, c("lit"), remove = FALSE)
 
 
 .onLoad = function(libname, pkgname) {
+  # Auto limit the max number of threads used by polars
+  if (
+    !(cargo_rpolars_feature_info()[["disable_auto_limit_max_threads"]] ||
+      isTRUE(getOption("polars.disable_auto_limit_max_threads"))) &&
```

Comment:
Related to #713, here `disable_auto_limit_max_threads` must be set via the `options` function.
Functions in the package `polars` are not useful here, since the value must be set when Polars is invoked.